runconfig/parse.go

```diff
@@ -187,6 +188,13 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 			MemorySwap = parsedMemorySwap
 		}
 	}
+	if *flSwappiness < 0 || *flSwappiness > 100 {
+		return nil, nil, cmd, fmt.Errorf("invalid value:%d. valid memory swappiness range is 0-100", *flSwappiness)
+	}
+	// This is to ensure that we do not try to override swappiness value for default case
+	if *flSwappiness == 60 {
+		*flSwappiness = -1
+	}
```

Comment:
You don't need to check any specific value, as I said in thread comment, you already treated `--memory-swappiness` 's default value as 60, so if user didn't assign any value, the default would be 60, if user didn't use option `--memory-swappiness`, the value would be inherited from parent which is done by cgroup itself. Which makes sense to me.

So you don't need to anything here, just remove this block.